paparazzi.sh

```diff
@@ -144,8 +144,15 @@ case "$1" in
         ./configure --prefix=/usr
         make -j $N_CORES
         sudo make install
+        # Do some linking
+        sudo rm /usr/lib64/libfreetype.so
```

Comment:
Sure, I have previously mentioned how this shell script is great but at the same time very risky with the kind of sudo privileges this gets. :D. Good luck.